src/include/disks.h

```diff
@@ -20,6 +20,7 @@
 #endif
 
 #include <stdint.h>
+#include <sys/types.h>
```

Comment:
The header uses loft_t, and it's defined in sys/types.h. Currently that header has been pulled in somewhere else and so compilation is fine,. If you want to reuse disks.h elsewhere like I did to make a standalone program to print out the BPB fields, then loft_t is undefined.